src/main.ts

```diff
@@ -1,11 +1,9 @@
-let s: HTMLScriptElement;
+function injectScript(file: string) {
+	const s: HTMLScriptElement = document.createElement("script");
+	s.src = chrome.runtime.getURL(file);
+	s.onload = () => s.remove();
+	(document.head || document.documentElement).append(s);
+}
```

Comment:
Made it simpler to inject scripts since there was just 2 statements that were the same